xbmc/GUIInfoManager.cpp

```diff
@@ -1277,377 +1763,709 @@ const infomap network_labels[] = {{ "isdhcp",            NETWORK_IS_DHCP },
+///     <p><hr>
+///     @skinning_v17 **[New Infolabel]** \link MusicPlayer_Property_Role_Orchestra `MusicPlayer.Property(Role.Orchestra)`\endlink
+///     <p>
+///   }
+///   \table_row3{   <b>`MusicPlayer.Property(Role.Lyricist)`</b>,
+///                  \anchor MusicPlayer_Property_Role_Lyricist
+///                  _string_,
+///     @return The name of the person who wrote the lyrics of the selected song.
+///     <p><hr>
+///     @skinning_v17 **[New Infolabel]** \link MusicPlayer_Property_Role_Lyricist `MusicPlayer.Property(Role.Lyricist)`\endlink
+///     <p>
+///   }
+///   \table_row3{   <b>`MusicPlayer.Property(Role.Remixer)`</b>,
+///                  \anchor MusicPlayer_Property_Role_Remixer
+///                  _string_,
+///     @return The name of the person who remixed the selected song.
+///     <p><hr>
+///     @skinning_v17 **[New Infolabel]** \link MusicPlayer_Property_Role_Remixer `MusicPlayer.Property(Role.Remixer)`\endlink
+///     <p>
+///   }
+///   \table_row3{   <b>`MusicPlayer.Property(Role.Arranger)`</b>,
+///                  \anchor MusicPlayer_Property_Role_Arranger
+///                  _string_,
+///     @return The name of the person who arranged the selected song.
+///     <p><hr>
+///     @skinning_v17 **[New Infolabel]** \link MusicPlayer_Property_Role_Arranger `MusicPlayer.Property(Role.Arranger)`\endlink
+///     <p>
+///   }
+///   \table_row3{   <b>`MusicPlayer.Property(Role.Engineer)`</b>,
+///                  \anchor MusicPlayer_Property_Role_Engineer
+///                  _string_,
+///     @return The name of the person who was the engineer of the selected song.
+///     <p><hr>
+///     @skinning_v17 **[New Infolabel]** \link MusicPlayer_Property_Role_Engineer `MusicPlayer.Property(Role.Engineer)`\endlink
+///     <p>
+///   }
+///   \table_row3{   <b>`MusicPlayer.Property(Role.Producer)`</b>,
+///                  \anchor MusicPlayer_Property_Role_Producer
+///                  _string_,
+///     @return The name of the person who produced the selected song.
+///     <p><hr>
+///     @skinning_v17 **[New Infolabel]** \link MusicPlayer_Property_Role_Producer `MusicPlayer.Property(Role.Producer)`\endlink
+///     <p>
+///   }
+///   \table_row3{   <b>`MusicPlayer.Property(Role.DJMixer)`</b>,
+///                  \anchor MusicPlayer_Property_Role_DJMixer
+///                  _string_,
+///     @return The name of the dj who remixed the selected song.
+///     <p><hr>
+///     @skinning_v17 **[New Infolabel]** \link MusicPlayer_Property_Role_DJMixer `MusicPlayer.Property(Role.DJMixer)`\endlink
+///     <p>
+///   }
+///   \table_row3{   <b>`MusicPlayer.Property(Role.Mixer)`</b>,
+///                  \anchor MusicPlayer_Property_Role_Mixer
+///                  _string_,
+///     @return The name of the dj who remixed the selected song.
+///     <p><hr>
+///     @skinning_v17 **[New Infolabel]** \link MusicPlayer_Property_Role_Mixer `MusicPlayer.Property(Role.Mixer)`\endlink
+///     <p>
```

Comment:
Would be nice to add that MusicPlayer.Property(Role.any_custom_role) also works, where any_custom_role could be an instrument `violin` or some other production activity e.g. `sound engineer`. The roles listed (composer, arranger etc.) are standard ones but there are many possible. Music file tagging allows for the musicians and all other people involved in the recording to be added, Kodi will gathers and stores that data, and it is availlable to GUI. 

So maybe rather than a row each have one entry for Role.XXXXX with `composer`, `arranger` etc. as listed values

Reply:
This one I'd prefer to tackle later. MusicPlayer.Property() can be any property right? I think it is better to create a link to where those properties are defined. It doesn't make much sense to have a table row per property as the Infolabel is still the same

Reply:
Sure no problem, add as a @todo